src/game/entity/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+target_sources(app PRIVATE 
+dev/red_square.cpp
+dev/blue_terrain.cpp
+dev/controllable.cpp
+)
```

Comment:
and a question, why not a `CMakeList.txt` inside `./src/game/entity/dev` and directly the names of the `.cpp`, without `dev/`?

Reply:
See 0fc88612472d3e788f89144e0840b55a9e7f39e1
Mostly did not bother doing it properly becaue it was a dev file, but there is proper CMake stuff now.
Also added a todo to fix it with target type (ie do not build dev in release)